### ISSUE TITLE
TypeError: Cannot read properties of null (reading 'boot')

### ISSUE THREAD

**Issue by ebullienc3** [Tenclea/YANG#149]
Running on Kali Linux.

TypeError: Cannot read properties of null (reading 'boot')
    at /snapshot/YANG/app.js:167:61
    at Object.<anonymous> (/snapshot/YANG/app.js:229:3)
    at Module._compile (pkg/prelude/bootstrap.js:1926:22)
    at Module._extensions..js (node:internal/modules/cjs/loader:1166:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Module._load (node:internal/modules/cjs/loader:834:12)
    at Function.runMain (pkg/prelude/bootstrap.js:1979:12)
    at node:internal/main/run_main_module:17:47
    
    [https://prnt.sc/RRybqIgfAhsU](https://prnt.sc/RRybqIgfAhsU)

**Comment by Tenclea** [comment]
Your config file must be outdated. Try deleting it, or moving it to another directory, and let the program make a new one.

**Comment by ebullienc3** [comment]
I did that already.

Here is my config:

# The length of the codes to generate, can be one of 'short', 'long' or 'both'
# Short is 16 characters, long is 24, and both will randomly choose between 16 and 24 each time
code_length: short

# Auto redeems valid nitro codes
auto_redeem:
  # enable auto_redeem or not (true / false)
  enabled: false
  # The token to redeem the codes with
  token: "PUT_YOUR_TOKEN_HERE"

# Proxy-related settings
proxies:
  # Write working proxies to a file (true / false)
  save_working: true

  # Validate proxies before passing them to the checker (true / false)
  enable_checker: false

  # Download fresh proxies from the web (true / false)
  enable_scrapper: false

  # The maximum amount of proxies to download (if enabled), use '0' for no maximum.
  max_proxies_download: 1000

# The amount of codes to check at the same time
# The higher, the faster, but it may slow the generator down if it is set too high
threads: 50

# Receive webhook messages on start and when a valid code is found
webhook:
  # enable webhook messages or not (true / false)
  enabled: true

  # The webhook url
  url: MY WEBHOOK

  # Webhook notifications settings
  notifications:
  # Send a notification when the generator starts (true / false)
  boot: true

  # Send a notification when a valid code is found (true / false)
  valid_code: true

  # How often to send status updates to the webhook in seconds (use '0' for never)
  status_update_interval: 600

# Print additional information in the console (true / false)
debug_mode: false

**Comment by Tenclea** [comment]
Oh I see what happened. I will fix the code tomorrow
In the meantime you can add two additional spaces before the following line:

```
# Send a notification when the generator starts (true / false)
boot: true

# Send a notification when a valid code is found (true / false)
valid_code: true
```

**Comment by Tenclea** [comment]
Fixed in [v3.3.4](https://github.com/Tenclea/YANG/releases/tag/v3.3.4), thank you for your precious help :)
